ext/BijectorsDistributionsADExt.jl

```diff
@@ -0,0 +1,114 @@
+module BijectorsDistributionsADExt
+
+if isdefined(Base, :get_extension)
+    using Bijectors
+    using DistributionsAD:
+        TuringDirichlet,
+        TuringWishart,
+        TuringInverseWishart,
+        FillVectorOfUnivariate,
+        FillMatrixOfUnivariate,
+        MatrixOfUnivariate,
+        FillVectorOfMultivariate,
+        VectorOfMultivariate,
+        TuringScalMvNormal,
+        TuringDiagMvNormal,
+        TuringDenseMvNormal
+else
+    using ..Bijectors
+    using ..DistributionsAD:
+        TuringDirichlet,
+        TuringWishart,
+        TuringInverseWishart,
+        FillVectorOfUnivariate,
+        FillMatrixOfUnivariate,
+        MatrixOfUnivariate,
+        FillVectorOfMultivariate,
+        VectorOfMultivariate,
+        TuringScalMvNormal,
+        TuringDiagMvNormal,
+        TuringDenseMvNormal
+end
+
+using LinearAlgebra
```

Comment:
This is not OK - an extension may only import from the main package or its associated weak dependency/dependencies. Even standard libraries must not be loaded directly.

Reply:
Ah, thanks! I just copy pasted from the compat files. Will change once I get back to computer